packages/containers/src/HeaderBar/HeaderBar.sagas.test.js

```diff
@@ -0,0 +1,58 @@
+import HeaderBarDefaultSaga, { fetchProducts, handleOpenProduct } from './HeaderBar.sagas';
+
+describe('HeaderBar sagas', () => {
+	describe('fetchProducts', () => {
+		it('should fetch HeaderBar products', () => {
+			const httpResponse = { response: { ok: true }, data: 'foo' };
+			const action = { payload: { url: 'url', lang: 'lang', env: 'env' } };
+
+			const gen = fetchProducts(action);
+
+			gen.next(); // Toggle fetching flag (enable)
+			const effect = gen.next().value; // HTTP call
+			expect(effect.CALL).toBeDefined();
+			gen.next(httpResponse); // Toggle fetching flag (disable)
+			gen.next(); // Update CMF collections
```

Comment:
Working on improving assertions right now, but do we really need to ensure that no store update action has been dispatched since we're already verifying every yield of the generator and checking that it has reached its end (`expect(done).toBe(true)`) ?

Reply:
You're right no need to if you have the check on `done`